Editing notes using the github.dev editor.md

```diff
@@ -0,0 +1,47 @@
+---
+aliases: 
+- 
+tags:
+- seedling
+publish: true
+---
+
+# Editing notes using the github.dev editor
+
+%% Add a description below this line. It doesn't need to be long: one or two sentences should be a good start. %%
+
+This guide will cover the basics of using the github.dev editor for the [[Obsidian Hub]]. This editor is great when you are making quick edits that don't need any Obsidian-specific functionality.
+
+## Opening the github.dev editor
+
+To open the editor you'll usually have two options: 
+1. On the Hub's publish site, each note has an "Edit in GitHub" button in it's footer
```

Comment:
Maybe worth linking to the Hub publish site and the Hub repo from these two bullet points... Many will not know what they mean, I feel.

Reply:
I've added those links.